File: Python/344.反转字符串.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        n = len(s)
        if n == 0:
            return
        for i in range(n/2):
            temp = s[i]
            s[i] = s[n-i-1]
            s[n-i-1] = temp
        return

File: CPP/344.反转字符串.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int n = s.size();
        if(n == 1 || n == 0){
            return ;
        }
        for(int i = 0; i < n/2; i++){
            char temp = s[i];
            s[i] = s[n-1-i];
            s[n-1-i] = temp;
        }
        return ;
    }
};